### ISSUE TITLE
Lumberjack doesn't do anything

### ISSUE THREAD

**Issue by NascostoClub** [ldtteam/minecolonies#5457]
# Prerequisites

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies [Wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**
- [x] This is an issue and not a feature request. Please create all feature requests [here](https://github.com/ldtteam/minecolonies-features/issues).

### Context

- Minecraft Version: 15.2
- MineColonies Version: minecolonies-0.13.59-ALPHA-universal

### Expected behavior

Lumberjack removes leaves and chops down tree

### Actual behaviour

Lumberjack is blocked by leaves covering the wood and stays there staring at the leaves forever

### Steps to reproduce the issue

Get a mod such as biomes of plenty where there are some trees that have leaves covering the trees wood.

**Comment by hron84** [comment]
Some updates:

 - You do not need Biomes o Plenty for reproducing this bug. Lumberjack cannot reach logs through leaves even with vanilla tress (sometimes oak can generate leaves at the level of the second log and because Lumberjack is 2 blocks high, cannot reach the trunk)
 - Lumberjack can break leaves and chop trees in the latest Beta version. So it's basically a regression since that.
 - Also pathfinding is somehow broken in Alphas for Lumberjack. He/She cannot see a whole forest behind his/her hut, I had to use the tool to select it.

**Comment by Raycoms** [comment]
I pushed a fix for this.

**Comment by NascostoClub** [comment]
Which version has the fix?

**Comment by ravenbuilder934** [comment]
The latest one should have it, others might as well
